### DISCUSSION TITLE
Can't use Global Css

### DISCUSSION THREAD

**Discussion by swpowe** [vercel/next.js#31170] [Help] [answered]
I'm just starting out trying to get familiar with next and currently I've run into a problem with global styles. I'm getting an error trying to import global.css from within my pages/_app.js. I'm just trying to walk through the basic intro tutorial so I'm not doing anything complex. Suggestions?

**Comment by relativepathson** [comment]
@swpowe  What error message do you get?

**Reply by swpowe** [reply]
Apologies, I should have provided more information.
![image](https://user-images.githubusercontent.com/24882933/140971825-4a19cb8f-3388-437b-bc9b-ef0d20679418.png)

**Reply by swpowe** [reply]
I've created my _app.js in the pages directory but when I try to import my global styles ( [as the walk through suggests](https://nextjs.org/learn/basics/assets-metadata-css/global-styles) ) I get the above error.

**Reply by relativepathson** [reply]
@swpowe I was unable recreate the above error. Please delete the **_app.js** file, then follow the steps in the walk through.

**Comment by swpowe** [comment]
I’ve tried on two different computers and I’ve deleted and started over multiple times. Could there be a conflict with versions of something? Any suggestions on what I could check? Thanks

**Comment by relativepathson** [comment]
@swpowe It’d be helpful if you could push your code to GitHub and link to it so that I can take a look. There should be no conflict if you used the **npx create-next-app <your-project-name>** command to bootstrap your app.

**Comment by swpowe** [comment, marked as answer]
I’ll push it a bit later. My code consists of the walk through and nothing else. It’s really basic. I was just trying to do the walk through posted on the home page and ran into this issue early on on both machines. 
I’ll take a look again later tonight and push something.  Thanks for your help.

**Reply by swpowe** [reply]
Looks like I might have figured it out. Needed up update webpack.
